plugins/puppetdb/README.md

```diff
@@ -0,0 +1,44 @@
+# PuppetDB Plugin
+
+A frontend plugin to integrate PuppetDB with Backstage. When combined with the
```

Comment:
Am I understanding this correctly: that without using the `PuppetDbEntityProvider` this plugin won't work? If that is true then I might want to call that out in bold or make that more obvious

Reply:
The requirement is to have `puppet.com/certname` annotation set for Resource kind of entities. This is done automatically by PuppetDbEntityProvider, but it can be done in some other way as well.

I've updated the README to include a new `Prerequisites` section and have elaborated more on the need of this annotation. Hope it makes it more clear.